megamek/build.gradle

```diff
@@ -291,6 +291,7 @@ ${project.ext.jvmOptions.join('\n')}
 
 windowsDistZip {
     description = 'Creates Windows distribution packaged as a zip archive'
+    dependsOn stageFiles
```

Comment:
This is missing if you just want to make a Windows build -- I suspect it wasn't a problem as typically when zips are made, they are all made and some other step invokes the `stageFiles` step.